templates/common/page_end.html

```diff
@@ -65,7 +65,7 @@ <h2 class="typeface-a color-a">Support</h2>
       $if active_users_string:
         ${active_users_string}.
         <br>
-      ${SHA} &middot; Copyright &copy; 2012–2016 Weasyl LLC
+      ${SHA} &middot; Copyright &copy; 2012–${current_year} Weasyl LLC
```

Comment:
`${arrow.now().year}` should work and avoid passing more variables to the template.